Jropp/jason-ropp-js-iobp
# User Sort Mini Project: Bug Fix - Resort on Save

## What It Does

Causes cards to resort on user save. 

Before, if a users department (or other sorted category) was changed, it did not immediately move into it's new category with the other users. 

- I refactored the sort function to be more universal
- Created an intermediary function for the dropdown menu to interface with the more universal function.
- I made a call to the sort function for every time users are updated.

## How To Test

Sort users by a category and then edit that category of the user. For example, sort users by Department, then change a users department. They should immediately be moved into that department.

## Notes/Caveats

I am undecided whether or not to collapse the user card after it is saved and relocated. I'm open to suggestions.

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:

## Dependencies
- [ ] #115   
- [ ] #117
